src/driver.ts

```diff
@@ -551,13 +554,18 @@ Please install or configure a preferred generator, or update settings.json or yo
     const expanded_flags = await Promise.all(expanded_flags_promises);
     log.trace('CMake flags are', JSON.stringify(expanded_flags));
 
+    // This resets for the cms-driver the client
+    // clears the configuration of cmake server
+    await this._refreshExpansions();
```

Comment:
Could you put a better explanation of why this call was moved here? I don't quite understand the purpose of it.

Reply:
Somethings I should go to bed before commits. 
The 'not configure yet' throws an error after restart of cmake client. 
The refreshexpansions triggers such a restart. If I move this line in front of the configure step then the configure step use the same expansions like the build step. It ensures the configure/codemodel/cmakeInputs are present in the build step and consistent. 

Does this descript help more? I would remove the sentence.

Reply:
Done